pkg/operator/ceph/csi/version.go

```diff
@@ -25,14 +25,14 @@ import (
 )
 
 var (
-	//minimum supported version is 3.8.0
-	minimum = CephCSIVersion{3, 8, 0}
+	//minimum supported version is 3.9.0
+	minimum = CephCSIVersion{3, 9, 0}
 	//supportedCSIVersions are versions that rook supports
-	releasev390 = CephCSIVersion{3, 9, 0}
+	releasev3100 = CephCSIVersion{3, 10, 0}
```

Comment:
How about dropping the patch from the var name? No need for the name to have it IMO.
```suggestion
	releasev310 = CephCSIVersion{3, 10, 0}
```